pkg/apiserver/handlers.go

```diff
@@ -139,6 +139,13 @@ func tooManyRequests(req *http.Request, w http.ResponseWriter) {
 func RecoverPanics(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		defer runtime.HandleCrash(func(err interface{}) {
+			if e, ok := err.(error); ok {
```

Comment:
I can do that too. but i am not sure if there are other cases that we want to log panics and handle it from timeout handler? Not all panics we throw from timeout handler are expected, right?